cg/cli/clean.py

```diff
@@ -133,6 +132,39 @@ def scoutauto(context, days_old: int, yes: bool = False, dry_run: bool = False):
         context.invoke(scout, bundle=bundle, yes=yes, dry_run=dry_run)
 
 
+@clean.command("hk-past-files")
+@click.option("-c", "--case-id", type=str)
+@click.option("-t", "--tags", multiple=True)
+@click.option("-y", "--yes", is_flag=True, help="skip checks")
+@click.option("-d", "--dry-run", is_flag=True, help="Shows cases and files that would be cleaned")
+@click.pass_context
+def hk_past_files(context, case_id, tags, yes, dry_run):
+    """ Remove files found in older housekeeper bundles """
+    if case_id:
+        cases = [context.obj["db"].family(case_id)]
+    else:
+        cases = context.obj["db"].families()
+    for case in cases:
+        case_id = case.internal_id
+        last_version = context.obj["hk"].last_version(bundle=case_id)
+        LOG.info("Searching %s bundle for outdated files", case_id)
+        hk_files = []
+        for tag in tags:
+            hk_files.extend(context.obj["hk"].get_files(bundle=case_id, tags=[tag]))
+        for hk_file in hk_files:
+            file_path = Path(hk_file.full_path)
+            if hk_file.version_id == last_version.id:
```

Comment:
I don't know if it can happen but if a file is part both of the latest bundle version AND a previous one we could end up removing it?

Reply:
That's a good question. I haven't thought of that, and just assumed that is never the case. But I think you are right, two file objects in hk could point to the same path... I will try to look into that!

Reply:
I have no idea if this actually will occur, but I've added a check in the last commit that would prevent removing such files. Maybe you can have a look at it @patrikgrenfeldt

Reply:
super!